pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -483,6 +485,14 @@ static void validateDecoder(StreamConfig streamConfig) {
     }
   }
 
+  public final static Set<AggregationFunctionType> AVAILABLE_CORE_VALUE_AGGREGATORS = Set.of(
```

Comment:
Let's use `EnumSet` here, and reformat the change

Reply:
done